tests/test_rebase_consensus.py

```
@@ -0,0 +1,896 @@
+            models=["m1", "m2"], consensus_threshold=0.8, prompt_templates={"m1": "tmpl"}
+        )
+        mock_cfg.llm = LLMConfig(cli_timeout=600)
+        mock_load.return_value = mock_cfg
+        models, threshold, templates, timeout = _load_consensus_config(Path("/fake"))
+        assert models == ["m1", "m2"]
+        assert threshold == 0.8
+        assert templates == {"m1": "tmpl"}
+        assert timeout == 600.0
+
+    @patch("sova.config.loader.load_config", side_effect=Exception("bad config"))
+    def test_returns_defaults_on_failure(self, _mock: MagicMock) -> None:
```

Ruff (0.16.1)

[warning] Fixture `_mock` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)